client/utils/evalAuth.js

```diff
@@ -0,0 +1,15 @@
+export const evalAuth = (data = {}) => {
+    const roles = data.roles || [];
+
+    return {
+        // calculated booleans
+        isAdmin: roles.includes('ADMIN') || false,
```

Comment:
I don't think we need the `|| false` part!

Should it be `roles && roles.includes('ADMIN')` or is roles always going to be defined?

Reply:
This could just be `roles.includes('ADMIN')` actually since roles is always defined on L2.